doc/user_guide/transformations.rst

```diff
@@ -382,6 +382,15 @@ can be found in the API-specific sections).
 
 ####
 
+.. autoclass:: psyclone.psyir.transformations.OMPCriticalTrans
+    :members: apply
+    :no-index:
+
+.. note:: PSyclone does not perform any checks on code generated with
+          OMPCriticalTrans, and it's correct usage is up to the user.
```

Comment:
This seems duplicated because it is already mentioned inside the docstring embeded just above. So I would deleted it. However, I agree the ".. note::" syntax make it look more highlighted. Could you instead move this tag inside the OMPCriticalTrans docsring second paragraph.